presto-docs/src/main/sphinx/functions/ml.rst

```diff
@@ -0,0 +1,158 @@
+==========================
+Machine Learning Functions
+==========================
+
+The machine learning plugin provides machine learning functionality
+as an aggregation function. It enables you to train Support Vector Machine (SVM)
+based classifiers and regressors for the supervised learning problems.
+
+.. contents::
+    :local:
+    :backlinks: none
+    :depth: 1
+
+.. note::
+
+  The machine learning functions are not optimized for distributed processing.
+  The capability to train large data sets is limited by this execution of the
+  final training on a single instance.
+
+Feature Vector
+--------------
+
+To solve a problem with the machine learning technique, especially as a
+supervised learning problem, it is necessary to represent the data set
+with the sequence of pairs of labels and feature vector. A label is a
+target value you want to predict from the unseen feature and a feature is a
+A N-dimensional vector whose elements are numerical values. In Presto, a
+feature vector is represented as a map-type value, whose key is an index
+of each feature, so that it can express a sparse vector.
+Since classifiers and regressors can recognize the map-type feature
+vector, there is a function to construct the feature from the existing
+numerical values, :func:`features`::
+
+    SELECT features(1.0, 2.0, 3.0) features;
```

Comment:
Some other queries are missing `as` of column alias and following `;`. Could you add it to be consistent in this page?
```suggestion
    SELECT features(1.0, 2.0, 3.0) as features;
```

Reply:
or even better .. use AS since we try to standardize on uppercase keywords